### DISCUSSION TITLE
IOTA vesting reward not displayed after collecting

### DISCUSSION THREAD

**Discussion by hrikgrik** [iotaledger/firefly#8636] [Help & Support]
Hello,
I tried to collect vesting rewards from staking Assembly , but when clicked to collect, it is not updating, and the box is greyed out.It is showing unlocked but not displaying balance correctly.I tried to reinstall Firefly,and tried from start with no success.Please assist.Thanks in advance

**Comment by begonaalvarezd** [comment]
The unlocked funds are part of your balance as soon as they unlock so they automatically compute in your balance without the need for you to collect them. Collecting is a way to cleanup your profile of small outputs and spread funds across multiple addresses, making your balance more organized and helping the reducing ledger size

**Comment by hrikgrik** [comment]
Hello,

Thanks for your reply.The unlocked funds are visible but the balance is not
displaying unlocked funds for the whole 2024 .I tried reinstalling the app
multiple times with no success. .Please see attached photos.

Best regards

**Reply by begonaalvarezd** [reply]
i dont see any photos attached sorry. I suggest you join our discord https://discord.iota.org/ to get further assistance, there you might be willing to share us your address so we could explain you how to understand the history or send us some screenshots
